Sources/SignalRClient/TransportType.swift

```diff
@@ -15,9 +15,11 @@ public struct TransportType: OptionSet {
         self.rawValue = rawValue
     }
 
-    static let longPolling = TransportType(rawValue: 1 << 0)
-    static let serverSentEvents = TransportType(rawValue: 1 << 1)
-    static let webSockets = TransportType(rawValue: 1 << 2)
+    public static let longPolling = TransportType(rawValue: 1 << 0)
+    public static let serverSentEvents = TransportType(rawValue: 1 << 1)
```

Comment:
I don't think it can be removed since we need to handle negotiation responses that could contain it. However, I have made it private so that it can't be specified in the list of allowed transport types.